src/utils/text.ts

```diff
@@ -17,6 +18,11 @@ export function expandTabs(line: string, tabWidth: number): string {
     return expanded;
 }
 
+export function splitGraphemes(str: string): string[] {
+    const splitter = new GraphemeSplitter();
```

Comment:
I remember the singleton I used before.

Reply:
Oh huh, I totally missed that subtlety with it being an IIFE. Nice!